### PR TITLE
Add support for 'hand' and 'gesture' Leap Motion events

### PR DESCRIPTION
**Pull request by stewart**
Adds :hand: + :wave: support.

Unsure if this is the best way to add them, let me know if there's a possible refactoring I'm missing.

### CODE REVIEW

**Comment by deadprogram** [comment]
That looks about like what I expected. Example or 2 please?

**Comment by deadprogram** [comment]
Also, do we have this documented in the godoc?

File: platforms/leap/leap_motion_driver.go [hunk 1]
```diff
@@ -60,9 +62,19 @@ func (l *LeapMotionDriver) Start() (errs []error) {
...
 		var msg []byte
+		var frame Frame
 		for {
 			receive(l.adaptor().ws, &msg)
-			gobot.Publish(l.Event("message"), l.ParseFrame(msg))
+			frame = l.ParseFrame(msg)
+			gobot.Publish(l.Event("message"), frame)
+
+			for _, hand := range frame.Hands {
+				gobot.Publish(l.Event("hand"), hand)
```
Comment (zankich) [inline platforms/leap/leap_motion_driver.go]: Do we want an event for each hand or one event with all the hands?

**Comment by deadprogram** [comment]
This PR is no longer valid, being that it is against master instead of dev. Closing.

**Comment by deadprogram** [comment]
I merged directly into `dev` branch, as the tests appeared to be passing.

**Pull request closed without merging**